client/package.json

```diff
@@ -159,6 +159,7 @@
     "pretest": "npm run lint & npm run licchk",
     "test": "node ./node_modules/vscode/bin/test --verbose",
     "posttest": "node ./node_modules/.bin/istanbul check-coverage",
+    "systest": "node ./node_modules/vscode/bin/test --verbose",
```

Comment:
remove systest line here

Reply:
This needs to stay as otherwise the coverage check will run and will fail